contracts/token/ERC721Token.sol

```diff
@@ -4,21 +4,17 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
-import "@openzeppelin/contracts/utils/Counters.sol";
```

Comment:
![image](https://github.com/The-Poolz/Poolz-Helper/assets/68740472/2c833ba1-6e19-4f4e-8b95-dea8f119cb70)

https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4233